app/src/db/migrations/20200602065740_009-minesoperatorscreening-common-model-changes.js

```diff
@@ -0,0 +1,27 @@
+const PREFIX = require('../../forms/minesoperatorscreening/constants').PREFIX;
+
+exports.up = function(knex) {
+  return Promise.resolve()
+    .then(() => knex.schema.alterTable(`${PREFIX}_submission`, table => {
+      table.boolean('deleted').notNullable().defaultTo(false);
```

Comment:
Think we should have a deletedAt to pair along with this? Granted it's not a requirement atm, but it's in consideration of potential auditing needs in the future?

Reply:
I think we'd want deletedAd deletedBy for audit

Reply:
Didn't want to put one in because it makes it an anomaly as far as tables go.  Any audit tables we have will show who updated it to delete concisely.

Reply:
We have createdBy updatedBy, in each table so doesn't it seem intuitive to look for deletedBy next to those rather than in a triggered audit table?

Reply:
deleted is just an updated data value, so I think it's weird to have an updatedBy/At that isn't in updatedBy/At but in deletedBy/At. Nothing else would even get updated in that table.

Reply:
Can go for it I guess. Not used yet anyways. Do we have audit tables yet?

Reply:
i dont see any audit tables in comfort. the 'logged_actions' table was only added recently to the silviculture site

Reply:
No audit tables yet.  Was kind of waiting to see how it worked in SILV. first.
Anyway, it looks weird to me basically a userstamp/timestamp for updating a specific field.  the updatedAt will still get automatically updated anyway, and what are we going to do if we update the data back from deleted?  remove the deletedBy/deletedAt?  I think it's better to just have it absorbed in the audit tables (when they come)